src/components/charts/ServicesChart.tsx

```diff
@@ -7,17 +7,17 @@ export default function ServicesChart() {
 
   const serviceCategories = getSubcategories('services sociétaux')
 
-  const serviceSubcategories = serviceCategories.reduce(
-    (acc, category) => {
-      acc[category] = getSubcategories(category)
+  const serviceSubcategories = serviceCategories?.reduce(
+    (acc: DottedName[], category: DottedName) => {
+      acc.push(...(getSubcategories(category) ?? []))
       return acc
     },
-    {} as Record<DottedName, DottedName[]>
+    [] as DottedName[]
```

Comment:
No need to cast acc and initial value. And if `serviceCategories` is a `DottedName[]` you can remove the category type.

Reply:
👍